docs/cli-spec.md

```diff
@@ -0,0 +1,47 @@
+# Signing CLI specficiation
+
+### Inputs
+
+The parameters to the signing client are as follows:
+
+`sign code AzureKeyVault [options] <files>`
+
+Options:
+
+- `-o | --output` - The output file. If omitted, overwrites input. Must be a directory if multiple files are specified.
+- `-b | --base-directory` - Base directory for files to override the working directory.
```

Comment:
Could you be able to elaborate more what is this base directory? Is it related to `files`?

Reply:
it's for any parameter that takes a relative path for files